src/client.rs

```diff
@@ -418,7 +476,243 @@ impl Client {
             stream_name.as_ref(),
             group_name.as_ref(),
             options,
+            false,
+        )
+        .await
+    }
+
+    /// Connects to a persistent subscription group to $all stream.
+    pub async fn connect_persistent_subscription_to_all<GroupName>(
```

Comment:
so we need to decide on the naming convention as per RFC022

Reply:
if we don't want to wait too long, we could create another issue for this ?
I guess in the Rust world , this is not too bad to rename things , people are used to it ?

Reply:
I opened the issue there: #108